pyblish_qml/control.py

```diff
@@ -1162,7 +1157,7 @@ def on_next(result):
             if not self.data["state"]["is_running"]:
                 return on_finished()
 
-            if isinstance(result, StopIteration):
+            if isinstance(result, Exception):
```

Comment:
This probably won't work. Now, whenever there is *any* exception, including actual faults or bugs, it will think the iterator is finished and exit successfully.

StopIteration was only ever provided if the iterator finished successfully. If we can't return a StopIteration, we need to pass something else, to differentiate an error from a success.

Reply:
Reverted